newton/_src/geometry/sdf_contact.py

```
@@ -47,6 +47,27 @@ class EdgeCullResult:
+@wp.func
+def _query_mesh_face_normal(
+    mesh_id: wp.uint64,
+    point_local: wp.vec3,
+    max_dist: float = 1000.0,
+) -> wp.vec3:
+    """Return the outward face normal of the closest triangle on a mesh.
+
+    Falls back to the point-to-center direction when the query misses.
+    The returned vector is in mesh-local space and is normalized.
+    """
+    face_index = int(0)
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)